Phonebook.Source.PeopleSoft/src/Phonebook.Source.PeopleSoft/Controllers/BuildingController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Phonebook.Source.PeopleSoft.Models;
+
+namespace Phonebook.Source.PeopleSoft.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class BuildingController : ControllerBase
+    {
+        public ModelContext Context { get; }
+
+        public BuildingController(ModelContext context)
+        {
+            Context = context;
+        }
+        // GET: api/Building
```

Comment:
Can you add valid XML markdown for integration of swashbuckle? 
Maybe we can also add swashbuckle in this pr or later on.

Reply:
--> Issue: #585